package.json

```diff
@@ -31,20 +24,26 @@
     "chromatic": "^6.11.4",
     "husky": "8.0.1",
     "lint-staged": "^13.0.2",
-    "patch-package": "^6.5.0",
     "prettier": "2.7.1",
+    "tsx": "^3.9.0",
     "turbo": "1.3.1",
     "typescript": "4.7.4"
   },
   "engines": {
     "node": "16",
-    "yarn": "1.22"
+    "pnpm": "7.18",
+    "yarn": "This project is configured to use pnpm"
```

Comment:
out of curiosity - what this is for? 🤔

Reply:
It will show the message when user try to run yarn without corepack enabled